models/engine/db_storage.py

```diff
@@ -74,3 +74,12 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ Get a object using class name and id """
+        dic = self.all(cls)
+        return dic["{}.{}".format(cls.__name__, id)]
```

Comment:
I'll suggest make the function get() to work with the cls as an string or as a class itself,  because the method all() manage that.